plan/physical_plan_builder.go

```diff
@@ -665,7 +690,23 @@ func (p *Aggregation) handleFinalAgg(x physicalDistSQLPlan, childInfo *physicalP
 	return info
 }
 
-func (p *Aggregation) handleHashAgg() (*physicalPlanInfo, error) {
+// convert2PhysicalPlanCompleteHash converts logical aggregation to complete hash aggregation *physicalPlanInfo.
```

Comment:
logical aggregation to complete hash aggregation *physicalPlanInfo. ->
the logical aggregation to the complete hash aggregation *physicalPlanInfo.